encryption-at-rest.md

```diff
@@ -17,7 +17,7 @@ Also from v4.0.0, BR supports S3 server-side encryption (SSE) when backing up to
 The current version of TiKV encryption has some drawbacks that we are working actively to address in the future versions.
 
 * When a TiDB cluster is deployed, the majority of user data is stored in TiKV nodes, and that data will be encrypted when encryption is enabled. However, a small amount of user data is stored in PD nodes as metadata (for example, secondary index keys used as TiKV region boundaries). As of v4.0.0, PD doesn't support encryption at rest. It is recommended to use storage-level encryption (for example, file system encryption) to help protect sensitive data stored in PD.
-* As of v4.0.0, TiFlash doesn't support encryption at rest. When deploying TiKV with TiFlash, data stored in TiFlash is not encrypted.
+* TiFlash supports encryption at rest starting from v4.0.5. For details, refer to [Encryption at Rest for TiFlash](/encryption-at-rest.md#encryption-at-rest-for-tiflash-new-in-v405). When deploying TiKV with TiFlash before v4.0.5, data stored in TiFlash is not encrypted.
```

Comment:
```suggestion
* TiFlash supports encryption at rest since v4.0.5. For details, refer to [Encryption at Rest for TiFlash](#encryption-at-rest-for-tiflash-new-in-v405). When deploying TiKV with TiFlash earlier than v4.0.5, data stored in TiFlash is not encrypted.
```